scripts/example-analysis.jl

```diff
@@ -0,0 +1,29 @@
+# First, read in the output energies and iterations as a DataFrame
+df = read_output("scripts/LJ_111_p08_output_df.csv")
+
+# Calculate the ω factors
+ωi = ωᵢ(df.iter, 640)
+
+# Shift the energies to be positive
+Ei = df.emax .- minimum(df.emax)
+# Specify the temperatures that we are interested in
+Ts = collect(10:1:2000)
+# Define the Boltzmann constant
+kb = 8.617333262e-5 # eV/K
+# Calculate the inverse temperatures
+β = 1 ./(kb.*Ts)
+# Define the degrees of freedom, which is 3×8 for the 8-particle system
+dof = 24
+
+# Calculate the partition functions for each temperature
+zs = [partition_function(b, ωi, Ei) for b in β]
+
+# Calculate the internal energies for each temperature
+u = [internal_energy(b, ωi, Ei) for b in β]
```

Comment:
We might want to be conscious about writing extensive thermodynamic properties in lower vs. upper case. Sometimes, lowercase refers to the molar property.

Reply:
Updated!